src/ui/components/NetworkMonitor/HeaderGroups.tsx

```diff
@@ -32,7 +26,7 @@ export function HeaderGroups({
               const { key, ...columnProps } = column.getHeaderProps();
               const isLastColumn = index === headerGroup.headers.length - 1;
               return (
-                <div className={classNames("p-1", styles[column.id])} {...columnProps} key={key}>
+                <div key={key} className={classNames("p-1", styles[column.id])} {...columnProps}>
```

Comment:
FIxes `key should be unique` warning showing up in the console. Not sure how having the props in the beginning works though

Reply:
Doesn't look like this should have changed anything, since `columnProps` (the value being spread) had `key` pulled out of it explicitly:
```js
const { key, ...columnProps } = column.getHeaderProps();
```

Reply:
Yeah it does have the `key` pulled out. The warning here I saw is about key not being unique(but they are actually unique, I confirmed that), not sure how putting it in the beginning fixes it. I don't have a solid answer here, but I tried to move it to the beginning and it did work.